### PR TITLE
Style: full-height code blocks

### PR DESCRIPTION
**Pull request by thockin**
Also whitespace

### CODE REVIEW

**Comment by caesarxuchao** [comment]
The css is not changed but the white space?

File: _sass/_documents.scss [hunk 1]
```diff
@@ -5,90 +5,97 @@ div #doccontent {
...
+        thead {
+            color: black;
+        }
+    }
 
-	p {
-		padding-bottom: 10px;
-		padding-top: 10px;
-	}
+    pre {
```
Comment (thockin) [inline _sass/_documents.scss]: @caesarxuchao other than whitespace, this changed.

Comment (thockin) [inline _sass/_documents.scss]: you can see it here:

https://github.com/GoogleCloudPlatform/kubernetes/pull/11504/files?w=0

**Comment by bgrant0607** [comment]
Needs rebase

**Comment by bgrant0607** [comment]
cc @krousey

**Comment by caesarxuchao** [comment]
LGTM

**Pull request merged**